test/test_convert.py

```diff
@@ -197,8 +192,7 @@ def test_embed_album_art(self):
         with open(os.path.join(image_path), 'rb') as f:
             image_data = f.read()
 
-        with control_stdin('y'):
-            self.run_convert()
+        self.run_convert(input='y')
```

Comment:
I'm thinkin that we should make these sort of changes in master if this is how we want the API to behave.  It shouldn't be hard to emulate it right? alternatively, we make attempt to make click act like the original.